web-admin/src/features/authentication/AvatarButton.svelte

```diff
@@ -1,17 +1,18 @@
 <script lang="ts">
   import { page } from "$app/stores";
   import {
-    Popover,
-    PopoverButton,
-    PopoverPanel,
-  } from "@rgossiaux/svelte-headlessui";
-  import { MenuItem } from "@rilldata/web-common/components/menu";
-  import Menu from "@rilldata/web-common/components/menu/core/Menu.svelte";
-  import { createPopperActions } from "svelte-popperjs";
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuItem,
```

Comment:
This is a nit, but I think `import * as DropdownMenu` and then accessing `Content`, `Root`, `Trigger`, etc. keeps this a lot cleaner.

Reply:
Agreed, and just updated. I was relying on VSCode auto-imports, so I'm curious – would you do this `import *` by hand, or do you happen to have a VSCode trick?